src/main/application-update/check-for-updates/process-checking-for-updates.injectable.ts

```diff
@@ -86,19 +82,6 @@ const processCheckingForUpdatesInjectable = getInjectable({
           broadcastChangeInUpdatingStatus({
             eventId: "download-for-update-failed",
           });
-
-          return;
-        }
-
-        const userWantsToInstallUpdate = await askBoolean({
-          title: "Update Available",
-
-          question: `Version ${version} of Lens IDE is available and ready to be installed. Would you like to update now?\n\n` +
-            `Lens should restart automatically, if it doesn't please restart manually. Installed extensions might require updating.`,
-        });
-
-        if (userWantsToInstallUpdate) {
-          quitAndInstallUpdate();
```

Comment:
Does something happen instead now?

Reply:
Yes, https://github.com/lensapp/lens/pull/5445 covers the quit and update case.